crates/iota/src/client_ptb/ptb.rs

```diff
@@ -173,19 +173,18 @@ impl PTB {
             Ok(x) => x,
         };
 
-        // get all the metadata needed for executing the PTB: sender, gas, signing tx
-        let gas = program_metadata.gas_object_id.map(|x| x.value);
-
-        let sender = match program_metadata.sender {
-            Some(sender) => sender.value.into_inner().into(),
-            // the sender is the gas object if gas is provided, otherwise the active address
-            None => match gas {
-                Some(gas) => context
-                    .get_object_owner(&gas)
-                    .await
-                    .map_err(|_| anyhow!("Could not find owner for gas object ID"))?,
-                None => context.active_address()?,
-            },
+        // TODO (amnn): support multiple gas objects
```

Comment:
For this one and the others, maybe remove the parentheses for now

Reply:
https://github.com/iotaledger/iota/pull/7881/commits/c0834eaba246b4fca302cc22705f1425bbfb7692